fineract-provider/src/main/java/org/apache/fineract/useradministration/service/RoleReadPlatformServiceImpl.java

```diff
@@ -25,19 +25,15 @@
 import org.apache.fineract.portfolio.self.registration.SelfServiceApiConstants;
 import org.apache.fineract.useradministration.data.RoleData;
 import org.apache.fineract.useradministration.exception.RoleNotFoundException;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.jdbc.core.RowMapper;
-import org.springframework.stereotype.Service;
 
-@Service
 public class RoleReadPlatformServiceImpl implements RoleReadPlatformService {
```

Comment:
Please cleanup with Lombok statements, see previous comments (log and constructor).

Reply:
I bet you RoleMapper is used only once (here) or not more than a handful of places in the whole source code... and that we do the same thing (instantiating it) everywhere in the exact same manner. If that is true then we can provide this somewhere as a ready to use injectable dependency in a Java configuration class for example.

Reply:
Let's leave it like this for now... not that important.